### ISSUE TITLE
stable version of gramine readthedocs is not updated with changes done as part of v1.8 release

### ISSUE THREAD

**Issue by anjalirai-intel** [gramineproject/gramine#2066]
### Description of the problem

The Gramine stable documentation on readthedocs is not updated with the changes done as part of v1.8 release

There are several other sections that need attention, but here are a few examples:

**latest version:**
1. CMake is listed as a required package.
Required Packages
https://gramine.readthedocs.io/en/latest/devel/building.html#required-packages
```
sudo apt-get install -y cmake libprotobuf-c-dev protobuf-c-compiler \
    protobuf-compiler python3-cryptography python3-pip python3-protobuf
```
2. Ubuntu 24.04 is included in the supported versions.
Install Gramine Packages
https://gramine.readthedocs.io/en/latest/installation.html#install-gramine-packages

**stable version:**

1. In the Required Packages section, CMake is missing from the list.
Required Packages
https://gramine.readthedocs.io/en/stable/devel/building.html#required-packages
```
sudo apt-get install -y libprotobuf-c-dev protobuf-c-compiler \
    protobuf-compiler python3-cryptography python3-pip python3-protobuf
```
2. Ubuntu 24.04 support is missing in the Install Gramine section.
Install Gramine Packages
https://gramine.readthedocs.io/en/stable/installation.html#install-gramine-packages-1

All changes released as part of v1.8 are missing from the stable readthedocs. It should be updated.

### Gramine commit hash

e1405529111807ba2f3fad8fcb911b1e5d8f38d1

**Comment by kailun-qin** [comment]
> In the Required Packages section, CMake is missing from the list.

I think this is because our ReadTheDocs build for the `stable` branch is temporarily failing. cc @woju 

See https://gramine.readthedocs.io/en/v1.8/devel/building.html#required-packages which shows the expected.

> Ubuntu 24.04 support is missing in the Install Gramine section.

Given that the subsection text was updated post v1.8 release (the installation instructions are the same): https://github.com/gramineproject/gramine/commit/f49927190b06b752c906e1a2c2fec6db50a3709e, I don't know if it can still be fixed.

**Comment by woju** [comment]
Fixed in PR #2073, please see. Unfortunately, to rebuild `stable` version on RTD, we need to push a tag, so this means we need to do a release. Either 1.8.1 from a branch, where I backport the fix from #2073, or we wait for 1.9.

**Comment by mkow** [comment]
I think we can wait, it's a lot of effort to do a release (even minor).
